mcrouter/scripts/docker/container/ubuntu-packaged/usr/local/bin/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -x
+
+
+function write_default_config() {
+cat << EOF > /etc/mcrouter.json
+{
+  "pools": {
+    "A": {
+      "servers": [
+        "127.0.0.1:5001"
+      ]
+    }
+  },
+  "route": "PoolRoute|A"
+}
+EOF
+}
+
+function run_mcrouter() {
+        local PORT=$1
+        mcrouter \
+        --config file:/etc/mcrouter.json
+#        -p "${PORT}"
+}
+
+
+if [ -z "${MCROUTER_LISTEN_PORT}" ]; then
```

Comment:
You can probably replace this if statement if you set the default port in the dockerfile as an ENV var. Then to change the port you just override the ENV var during docker run.